themes/luan/luan.theme.sh

```diff
@@ -10,20 +10,23 @@ GIT_THEME_PROMPT_CLEAN=" ${_omb_prompt_bold_green}✓"
 GIT_THEME_PROMPT_PREFIX="(${_omb_prompt_olive}"
 GIT_THEME_PROMPT_SUFFIX="${_omb_prompt_normal})"
 
-RVM_THEME_PROMPT_PREFIX=""
-RVM_THEME_PROMPT_SUFFIX=""
+OMB_PROMPT_RVM_FORMAT="%s"
+OMB_PROMPT_VIRTUALENV_FORMAT='(%s) '
+OMB_PROMPT_SHOW_PYTHON_VENV=${OMB_PROMPT_SHOW_PYTHON_VENV:=false}
 
 function _omb_theme_PROMPT_COMMAND() {
-    dtime="$(clock_prompt)"
-    user_host="${_omb_prompt_green}\u@${_omb_prompt_teal}\h${_omb_prompt_normal}"
-    current_dir="${_omb_prompt_bold_navy}\w${_omb_prompt_normal}"
-    rvm_ruby="${_omb_prompt_bold_brown}$(_omb_prompt_print_ruby_env)${_omb_prompt_normal}"
-    git_branch="$(scm_prompt_info)${_omb_prompt_normal}"
-    prompt="${_omb_prompt_bold_green}\$${_omb_prompt_normal} "
-    arrow="${_omb_prompt_bold_white}▶${_omb_prompt_normal} "
-    prompt="${_omb_prompt_bold_green}\$${_omb_prompt_normal} "
+  local dtime="$(clock_prompt)"
+  local user_host="${_omb_prompt_green}\u@${_omb_prompt_teal}\h${_omb_prompt_normal}"
+  local current_dir="${_omb_prompt_bold_navy}\w${_omb_prompt_normal}"
+  local ruby_env python_venv
+  _omb_prompt_get_ruby_env && ruby_env="$_omb_prompt_bold_brown$ruby_env$_omb_prompt_normal "
+  _omb_prompt_get_python_venv
+  local git_branch="$(scm_prompt_info)${_omb_prompt_normal}"
+  local prompt="${_omb_prompt_bold_green}\$${_omb_prompt_normal} "
+  local arrow="${_omb_prompt_bold_white}▶${_omb_prompt_normal} "
+  local prompt="${_omb_prompt_bold_green}\$${_omb_prompt_normal} "
 
-    PS1="${dtime}${user_host}:${current_dir} ${rvm_ruby} ${git_branch}
+  PS1="${dtime}${python_venv}${user_host}:${current_dir} ${ruby_env}${git_branch}
```

Comment:
@Askaiant The `luan` theme already seems to have the ruby-env information in the later part after `${current_dir}`. It seems to me that there is no reason to include two similar segments in different positions in the prompt. Can we move the python-env information so that `${ruby_env}` and `${python_venv}` are located next to each other?

Reply:
Agree that that would be the most natural spot perhaps for this theme, this was kinda my own personal preference as I liked the default bash format of venv shown like [this](https://i.stack.imgur.com/3zUIW.png). Just to pick your brain a bit, what's your thought on adding venv on start of the prompt (like the default bash) for themes that haven't implemented the "show_python_venv"?

Reply:
Yes! I think it would be a good default to put it at the start of the prompt by default unless there are no conflicts or a design issue like this.

Anyway, I would like to first merge this PR and then pick up the other PRs https://github.com/ohmybash/oh-my-bash/pull/387, https://github.com/ohmybash/oh-my-bash/pull/373, https://github.com/ohmybash/oh-my-bash/pull/335, and https://github.com/ohmybash/oh-my-bash/pull/282 after adjustments. I think this is a good way to count the contributions of these people. I'd like to process the other themes after that if necessary.

Reply:
I have moved `${python_venv}` just before `${ruby_env}`